data/json/martialarts.json

```diff
@@ -1038,12 +1038,86 @@
     ],
     "weapons": [ "butterfly_swords", "long_pole" ]
   },
+  {
+    "type": "martial_art",
+    "id": "style_medievalpole",
+    "name": "Fior Di Battaglia",
+    "description": "Medieval Europe's martial techniques for fighting with polearms, great focus is given in countering one's opponent and knocking them down before landing a killing blow",
+    "initiate": [ "You set your feet apart and turn your torso slightly.", "%s sets their feet apart and turns their torso slightly." ],
+    "learn_difficulty": 6,
+    "arm_block": 99,
+    "leg_block": 99,
+    "static_buffs": [
+      {
+        "id": "medievalpole_static",
+        "name": "Stand Your Ground",
+        "description": "+2 blocks -1 dodge, halves your dodge and damage is reduced by your strength when blocking",
+        "strictly_melee": true,
+        "flat_bonuses": [ [ "block", "str", 1.0 ] ],
+        "mult_bonuses": [ [ "dodge", 0.5 ] ],
+        "bonus_dodges": -1,
+        "bonus_blocks": 2
+      }
+    ],
+    "onmove_buffs": [
+      {
+        "id": "medievalpole_move",
+        "name": "You moved",
+        "description": "Nulifies Stand Your Ground, don't move, stand there and counter!",
+        "strictly_melee": true,
+        "flat_bonuses": [ [ "block", "str", -1.0 ] ],
+        "mult_bonuses": [ [ "dodge", 1.0 ] ],
```

Comment:
Since this is a multiplicative bonus, does it have to be `2.0` to counter the `0.5` in `Stand Your Ground`?

Reply:
from what I understood and comparing with others since stand your ground has no duration the you moved debuff would override it... did it not?

Reply:
I haven't tested it, so I can't say. I imagine if they both apply, it would multiply them both (`1.0` and `0.5`) together and get `[ [ "dodge", 0.5 ] ]`.

Reply:
Nah I just tested it yesterday, when you stand still your dodge gets reduced but when you move, it goes back to normal, so it looks like I did something right, YAY!